.eslintrc

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
:small_blue_diamond: One minor non-blocker (can also ignore if not obvious) -- Is it possible to retain the names in the vendor chunks?

In https://github.com/OdyseeTeam/odysee-frontend/issues/2112, it was easy to detect which chunk was causing problems -- I just ignore the hash.

With the name [now gone](https://github.com/OdyseeTeam/odysee-frontend/pull/1902#issuecomment-1228070558), it might be troublesome to figure out which chunk failed.  Is there an option to retain the vender naming?  If not, doesn't matter -- not that important.

Reply:
Cool.  I'll have some time this weekend to take a look at this and see about this.

Reply:
@infinite-persistence I've been looking at this and want to check on a couple of things.

As for the dev server, yeah, webpack 5 (or, more specifically [webpack-dev-sefver](https://github.com/webpack/webpack-dev-server/) run in-memory. There is a `devMiddleware.writeToDisk` option, but that also removes some of speed benefit. Would you like that enabled? Additionally, it could be env configurable.

Regarding the simple hashes, what script are you using to generate that result? I'm seeing the following

When running: `yarn dev:web-server`
![image](https://user-images.githubusercontent.com/128739/189469589-3c98cf9c-b3a6-4845-84bd-c7666c1d002c.png)

And when running: `NODE_ENV=production yarn compile:web`
![image](https://user-images.githubusercontent.com/128739/189469684-3fd23d51-5270-4f11-8b90-e908f2c92141.png)

And of course, `dev:web` is in-memory. Though, I'm not certain why it's got node_modules as part of the name, it seems that the names are lining up on both sides, with the exception of the content hash, of course.

Reply:
Just wanted to swing back on this one just to be sure I've got all the information I need to wrap up any loose ends on this by this weekend, including the conflict. Let me know if there's any clarification you need.

Reply:
> _Would you like that enabled? Additionally, it could be env configurable._

Nope, knowing how to enable it later for debugging is sufficient. Thanks for investigating.

> _Regarding the simple hashes, what script are you using to generate that result?_

- I think I used `NODE_ENV=production yarn compile:web` to compare output with current `master`.
- But I tried again just now and it looks good (fully populated names).
    - However, it does not have `*node_modules*` like in your screenshot, so it's weird that we are seeing outputs.  But maybe not a big deal.

Reply:
@infinite-persistence I looked at it a bit this weekend, and I'm wondering if this has to do with the duplicate dev dependencies in the `web` directory, which requires a second install in the child directory.  I originally removed those, but put them back in after running into some weirdness.  I think it had to do with the fact that the `compile:web` went through a second node call to run webpack (`node --max_old_space_size=4096 ./node_modules/webpack/bin/webpack.js --config webpack.config.js`) vs calling it directly (`webpack --config webpack.config.js`), and my goal here is to modify as little as possible. However, in this case, it might be considered an arms-reach cleanup to avoid any potential double-install after branch switch confusions.

I've made the above change and also updated from master.  Take another look and let me know what you think.

Reply:
- The hash-only filenames came back.  But again, probably can live with it.
     - <img width="376" alt="image" src="https://user-images.githubusercontent.com/64950861/191174144-2b7ef9d2-9278-4217-9a49-eccbd31e25de.png">
- I'm checking with Jessop on the duplicate dependencies in `web`.  I think I asked once why we needed to maintain both sides with the same thing rather than inherit, but can't recall the answer.  Seems to build fine with the removal, though.

Reply:
Weird. Perhaps it's some node or yarn version thing? I'm currently on node 16 and yarn 1.22.

Would you like me to put the dependencies back? Other than that, any other todos here?

Reply:
All good for now, everything is functional :+1: